examples/gpu/eemumu_AV/SubProcesses/P1_Sigma_sm_epem_mupmum/check.cc

```diff
@@ -318,21 +318,17 @@ int main(int argc, char **argv)
     // 3b. Evaluate MEs on the device
     // 3c. Copy MEs back from device to host
 
-
```

Comment:
Was the removal of this blank line intentional?

Reply:
Yes, Doing some aesthetic cleanup (my own personal definition). At some point it would be nice to have som ecoding styles recommended, or even enforced by tools. For the moment it's "have a look every now and then and make it look good"!